styles.css

```diff
@@ -116,12 +120,100 @@ body {
   padding-top: 2rem;
 }
 
+/* project popup start here */
+.popup-container {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 10, 30, 0.5);
+  z-index: 9999;
+}
+
+.popup-content {
+  position: relative;
+  width: 343px;
+  height: 100vh;
+  background-color: #fff;
+  align-items: center;
+  padding: 12px 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin-left: auto;
+  margin-right: auto;
+  overflow-y: scroll;
+}
```

Comment:
- [ ] I have noticed that your `popup-content` has a fixed width. This breaks the responsiveness of the popup in different screen sizes. Kindly consider using the percentage width to match the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=u9NC5n6sAS6bpo65-0) 😎
- [ ] In addition, kindly ensure that the height of the popup content matches the Figma design. You may need to consider reducing the height accordingly ☺️.

Your design 😎 
![image](https://github.com/manzitresor/Portfolio/assets/99394374/3b282a7c-518b-47be-9f2d-649986d5e61a)

Figma design 😎 
![image](https://github.com/manzitresor/Portfolio/assets/99394374/9c2dd3ae-cbdb-48d1-a656-30c499193c49)